package.json

```diff
@@ -135,7 +135,7 @@
   "scripts": {
     "analyze": "source-map-explorer build/static/js/main.*",
     "build": "run-s build-css build-js",
-    "build-css": "sass -I ./node_modules src/:src/ --watch",
```

Comment:
`--watch` is pretty useful though

Reply:
`--watch` is still used for `watch-css`.  Before my change `build-css` did not contain `--watch`. I copy-pasted wrong which caused the build to fail because it was all waiting for the `build-css` to finish.